GVFS/GVFS.Common/ProductUpgraderInfo.cs

```diff
@@ -51,7 +53,7 @@ public void DeleteAllInstallerDownloads()
 
         public void RecordHighestAvailableVersion(Version highestAvailableVersion)
         {
-            string highestAvailableVersionFile = GetHighestAvailableVersionFilePath();
+            string highestAvailableVersionFile = GetHighestAvailableVersionFilePath(GVFSPlatform.Instance.GetDataRootForGVFSComponent(ProductUpgraderInfo.UpgradeDirectoryName));

```

Comment:
Call `GetUpgradesDirectoryPath()` here instead of `GVFSPlatform.Instance.GetDataRootForGVFSComponent(ProductUpgraderInfo.UpgradeDirectoryName)`?